docs/analyzer-rules/AZFW0010.md

```diff
@@ -0,0 +1,24 @@
+# AZFW0010: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00010|
```

Comment:
Why not AZFW0009? https://github.com/Azure/azure-functions-dotnet-worker/tree/main/docs/analyzer-rules

Reply:
9 already exists in our feature branch: https://github.com/Azure/azure-functions-dotnet-worker/tree/feature/sdk-type-binding/docs/analyzer-rules